src/x/instrument/build.go

```diff
@@ -131,13 +131,19 @@ func (b *buildReporter) Start() error {
 }
 
 func (b *buildReporter) report() {
-	scope := b.opts.MetricsScope().Tagged(map[string]string{
+	tags := map[string]string{
 		"revision":      Revision,
 		"branch":        Branch,
 		"build-date":    BuildDate,
 		"build-version": Version,
 		"go-version":    goVersion,
-	})
+	}
+
+	for k, v := range b.opts.CustomBuildTags() {
+		tags[k] = v
```

Comment:
any worry of overriding existing tags?

Reply:
I can actually see cases in which we'd want to override. For example, the only way to really set `Version` etc. right now is by passing `ldflags`, and some users may have other ways they want to inject such metadata.